packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -17,8 +17,8 @@ export interface ColorPickerProps extends Themeable {
    */
   onColorChange?: ColorPickerChangeHandler;
   enableNamedColors?: boolean;
-  children?: JSX.Element;
```

Comment:
This is not actually used in the Picker components just in the `colorPickerFactory` so I moved it there.